### PR TITLE
sphinx 2.2.10

### PR DESCRIPTION
**Pull request by alpaker**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
This needs rebased on `master`.

**Comment by alpaker** [comment]
@mikemcquaid Sorry about that.  Rebased.

File: Library/Formula/sphinx.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Sphinx < Formula
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/sphinx.rb]: Sorry, I didn't notice this before: this line needs remove as it's a new version. My apologises for the back-and-forth.

**Comment by alpaker** [comment]
@mikemcquaid Removed `revision 1` statement.

File: Library/Formula/sphinx.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
 class Sphinx < Formula
   desc "Sphinx is a full-text search engine"
   homepage "http://www.sphinxsearch.com"
-  revision 1
 
   stable do
-    url "http://sphinxsearch.com/files/sphinx-2.2.9-release.tar.gz"
-    sha256 "79bcb9fca069ba630fd71fb40fba05bb16e19b475906fb6ae026334d50a6bf3a"
+    url 'http://sphinxsearch.com/files/sphinx-2.2.10-release.tar.gz'
+    sha256 '054cb86e64bd48997d89386e1224d4405063d9857b2d7c33dc6cc1b9ef6df440'
```
Comment (bfontaine) [inline Library/Formula/sphinx.rb]: Please keep the double quotes.

Comment (mikemcquaid) [inline Library/Formula/sphinx.rb]: I fixed this when I pulled.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Comment by xu-cheng** [comment]
It's merged in https://github.com/Homebrew/homebrew/commit/04e30782beaf4ffed01c2ec743073acbb8d7a3dc

**Pull request closed without merging**